src/box/lua/console.lua

```diff
@@ -620,6 +620,9 @@ local function local_read(self)
             buf = buf:sub(0, #buf - #delim)
             break
         end
+        if discard_buffer == true then
+            return ''
+        end
```

Comment:
Shouldn't we check it before attempt to parse a command? It seems more logical to me, however since we always get `line == ''` at Ctrl+C I can't find any difference in the actual behaviour.

Reply:
Fixed.